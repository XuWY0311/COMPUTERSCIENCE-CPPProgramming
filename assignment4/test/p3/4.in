Iabcde
B
B
D
Ixyz
J
D
D
Iuvw
E
Iopq
B
B
E
Iijk
D