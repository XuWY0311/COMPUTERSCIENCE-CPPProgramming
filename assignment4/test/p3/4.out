 a b c d e
          ^
 a b c d e
        ^
 a b c d e
      ^
 a b c e
      ^
 a b c x y z e
            ^
 a b c x y z e
^
 b c x y z e
^
 c x y z e
^
 u v w c x y z e
      ^
 u v w c x y z e
                ^
 u v w c x y z e o p q
                      ^
 u v w c x y z e o p q
                    ^
 u v w c x y z e o p q
                  ^
 u v w c x y z e o p q
                      ^
 u v w c x y z e o p q i j k
                            ^
 u v w c x y z e o p q i j k
                            ^
